.gitignore

```diff
@@ -105,3 +105,9 @@ ENV/
 
 # pytest
 .pytest_cache/
+
+# Test data
+crank/tests/files/
```

Comment:
This test folder include a few input files for the test_stack to run, so I think it shouldn't be ignored here.

Reply:
You can still manually add more files here, but this will prevent accidentally adding many scratch files.